src/CompanyService.Models.Broker/Models/UserData.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+
+namespace LT.DigitalOffice.UserService.Models.Broker.Models
+{
+    public class UserData
+    {
+        public Guid Id { get; set; }
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+        public string MiddleName { get; set; }
+        public bool IsActive { get; set; }
+
+        public static UserData Create(Guid id, string firstName, string middleName, string lastName, bool isActive)
+        {
+            return new UserData
+            {
+                Id = id,
+                FirstName = firstName,
+                LastName = lastName,
+                MiddleName = middleName,
+                IsActive = isActive
+            };
```

Comment:
In userService exist model for this operation. I think you can use

Reply:
Do not understand